subprojects/packagefiles/glew/meson.build

```diff
@@ -23,45 +23,45 @@ project(
   'c',
   version : '2.2.0',
   license: ['MIT', 'BSD-3'],
-  meson_version : '>= 0.43.0'
+  meson_version : '>= 0.49.0'
 )
 
 cc = meson.get_compiler('c')
 host_system = host_machine.system()
 
-user_args = []
+dep_gl = dependency('gl', required: false)
+if not dep_gl.found()
+  dep_gl = dependency('opengl')
+endif
```

Comment:
That's why we have `ependency('gl', 'opengl')` now :)

Reply:
Hmm?

Reply:
We can give multiple names to `dependency()` and it will return the first found. But that requires fairly new Meson version, I think @eli-schwartz wants to keep wrapdb compatible with older Meson version. So let's ignore this.